pkg/controller/controller_test.go

```diff
@@ -2267,6 +2268,41 @@ var _ = ginkgo.Describe("CSIControllerService", func() {
 				}))
 			})
 
+			ginkgo.It("should successfully expand scsi volume when metro is enabled", func() {
+				clientMock.On("GetVolume", mock.Anything, validBaseVolID).Return(gopowerstore.Volume{
+					MetroReplicationSessionID: validSessionID,
+					Size:                      validVolSize,
+				}, nil)
+				clientMock.On("ModifyVolume",
+					mock.Anything,
+					mock.AnythingOfType("*gopowerstore.VolumeModify"),
+					validBaseVolID).
+					Return(gopowerstore.EmptyResponse(""), nil)
+				clientMock.On("GetVolumeGroupsByVolumeID", mock.Anything, validBaseVolID).Return([]gopowerstore.VolumeGroup{}, nil)
```

Comment:
We do not need VG mocks in expand.

Reply:
Ah yeah that was mocked for our older version of the code. Updated now